src/layer/__tests__/cache.ts

```diff
@@ -0,0 +1,74 @@
+import 'jest-setup';
+import axios from 'axios';
+import MockAdapter from 'axios-mock-adapter';
+import makeServiceWorkerEnv from 'service-worker-mock';
+
+import { constructFixtureFindTiles } from './fixtures.findTiles';
+
+const mockNetwork = new MockAdapter(axios);
+
+describe('Testing caching', () => {
+  beforeEach(() => {
+    Object.assign(global, makeServiceWorkerEnv());
+    jest.resetModules();
+  });
+
+  it('should fetch a request and cache it, where 2nd request is served from the cache', async () => {
+    jest.setTimeout(7000);
```

Comment:
Since we are not timing out on requests, this line should probably be removed (it is not needed and indicates that this is a long-running test, which it is not).

Reply:
I have removed all the `jest.setTimeout`